blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -51,7 +53,7 @@ public void getFolderPipelineTest() throws IOException {
         MockFolder folder = j.createFolder("folder1");
         Project p = folder.createProject(FreeStyleProject.class, "test1");
 
-        Map response = get("/organizations/jenkins/pipelines/folder1/test1");
+        Map response = get("/organizations/jenkins/pipelines/folder1/pipelines/test1");
```

Comment:
@michaelneale Noticed this was not the right URL, it was working by accident as folder API allows such call in limited sort of way. Why do you ask this question?